joblib/test/test_parallel.py

```diff
@@ -407,15 +474,15 @@ def test_batching_auto_multiprocessing():
 
 
 def test_exception_dispatch():
-    "Make sure that exception raised during dispatch are indeed captured"
+    """Make sure that exception raised during dispatch are indeed captured"""
     with raises(ValueError):
         Parallel(n_jobs=2, pre_dispatch=16, verbose=0)(
             delayed(exception_raiser)(i) for i in range(30))
 
 
 def test_nested_exception_dispatch():
-    # Ensure TransportableException objects for nested joblib cases gets
-    # propagated.
+    """Ensure TransportableException objects for nested joblib cases gets
```

Comment:
Don't change it back but I don't expect any consistency whatsoever in docstrings vs comments. Historically comments were preferred because nose use the docstring for the test name which many people including me found annoying. Not a problem any more with pytest.